docs/source/guides/models.mdx

```diff
@@ -1,217 +0,0 @@
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
--->
-
-# Neuron Model Inference
-
-*The APIs presented in the following documentation are relevant for the inference on [inf2](https://aws.amazon.com/ec2/instance-types/inf2/),
-[trn1](https://aws.amazon.com/ec2/instance-types/trn1/) and [inf1](https://aws.amazon.com/ec2/instance-types/inf1/).*
-
-`NeuronModelForXXX` classes help to load models from the [Hugging Face Hub](hf.co/models) and compile them to a serialized format optimized for
-neuron devices. You will then be able to load the model and run inference with the acceleration powered by AWS Neuron devices.
-
-## Switching from Transformers to Optimum
-
-The `optimum.neuron.NeuronModelForXXX` model classes are APIs compatible with Hugging Face Transformers models. This means seamless integration
-with Hugging Face's ecosystem. You can just replace your `AutoModelForXXX` class with the corresponding `NeuronModelForXXX` class in `optimum.neuron`.
-
-If you already use Transformers, you will be able to reuse your code just by replacing model classes:
-
-```diff
-from transformers import AutoTokenizer
--from transformers import AutoModelForSequenceClassification
-+from optimum.neuron import NeuronModelForSequenceClassification
-
-# PyTorch checkpoint
--model = AutoModelForSequenceClassification.from_pretrained("distilbert-base-uncased-finetuned-sst-2-english")
-
-+model = NeuronModelForSequenceClassification.from_pretrained("distilbert-base-uncased-finetuned-sst-2-english",
-+                                                             export=True, **neuron_kwargs)
-```
-
-As shown above, when you use `NeuronModelForXXX` for the first time, you will need to set `export=True` to compile your model from PyTorch to a neuron-compatible format.
-
-You will also need to pass Neuron specific parameters to configure the export. Each model architecture has its own set of parameters, as detailed in the next paragraphs.
-
-Once your model has been exported, you can save it either on your local or in the [Hugging Face Model Hub](https://hf.co/models):
-
-```python
-# Save the neuron model
->>> model.save_pretrained("a_local_path_for_compiled_neuron_model")
-
-# Push the neuron model to HF Hub
->>> model.push_to_hub(  # doctest: +SKIP
-...     "a_local_path_for_compiled_neuron_model", repository_id="my-neuron-repo", use_auth_token=True
-... )
-```
-
-And the next time when you want to run inference, just load your compiled model which will save you the compilation time:
-
-```python
->>> from optimum.neuron import NeuronModelForSequenceClassification
->>> model = NeuronModelForSequenceClassification.from_pretrained("my-neuron-repo")
-```
-
-As you see, there is no need to pass the neuron arguments used during the export as they are
```

Comment:
I think that we should add the snippet of loading pre-compiled models somewhere (don't feel like it should be in `export_model` neither), it might be obvious for us but better explain properly to users, export is necessary each time, and where compilation args are stored.

Reply:
sorry, missed that part.